code/android-core-library/src/main/java/com/adobe/marketing/mobile/Log.java

```diff
@@ -10,135 +10,123 @@
  */
 package com.adobe.marketing.mobile;
 
+import com.adobe.marketing.mobile.services.Logging;
+import com.adobe.marketing.mobile.services.ServiceProvider;
+
 /**
  * Logging class to handle log levels and platform-specific log output
  *
  * @author Adobe Systems Incorporated
  * @version 5.0
  */
-class Log {
-	private static LoggingService loggingService = null;
-	private static LoggingMode loggingMode = LoggingMode.ERROR;
-
-	static final String UNEXPECTED_NULL_VALUE = "Unexpected Null Value";
-	static final String UNEXPECTED_EMPTY_VALUE = "Unexpected Empty Value";
-	static final String INVALID_FORMAT = "Invalid Format";
+public class Log {
+    private static Logging loggingService = ServiceProvider.getInstance().getLoggingService();
```

Comment:
This will work for now. We should get the instance just before logging when we add a method to override Logging service.